resfulservice/src/api-docs/swagger-service.yaml

```diff
@@ -538,6 +615,308 @@ paths:
                   message:
                     type: string
                     example: An error occurred
+  
+  /curate/bulk:
+     post:
+      security:
+        - BearerAuth: [] 
+      summary: Creates bulk curations
+      description: Creates multiple curations that are uploaded puts in separates folders uploaded in a zip file
```

Comment:
Not sure if I'm exactly sure what this is saying, but maybe "Creates multiple curations that are uploaded in separate folders uploaded in a zip file" ?